Remove notice about redis disk resizing (#79)
vshn/appcat-user-docs: docs/modules/ROOT/pages/vshn-managed/redis/plans.adoc

    name: postgres-creds
----

> NOTE: When changing the disk size from a plan, it will restart the redis pods. Any subsequent disk resizes should not trigger restarts.

Reviewer: ```suggestion
NOTE: Changing the disk size from a plan will restart the Redis pods. Any further disk size changes should not trigger a restart.
```

Author: I've modified your suggestion slightly, what do you think?

Reviewer: approved

> 

Reviewer: Does it mean that any other disk size changes after the first one will not trigger restart?

Author: Yes.

If you have suggestions to re-phrase it better, please feel free to suggest :D

Reviewer: thinking..
does should mean that it could? :D

Author: I haven't observed it, but I'm also not 100% sure...

In my tests it never triggered a restart after the first time.
